ably/realtime_conn.go

```diff
@@ -18,27 +18,32 @@ var (
 // Conn represents a single connection RealtimeClient instantiates for
 // communication with Ably servers.
 type Conn struct {
-	details       proto.ConnectionDetails
-	id            string
-	serial        int64
-	msgSerial     int64
-	err           error
-	conn          proto.Conn
-	opts          *ClientOptions
-	state         *stateEmitter
-	stateCh       chan State
-	pending       pendingEmitter
-	queue         *msgQueue
-	auth          *Auth
-	onChannelMsg  func(*proto.ProtocolMessage)
-	onStateChange func(State)
+	details      proto.ConnectionDetails
+	id           string
+	serial       int64
+	msgSerial    int64
+	err          error
+	conn         proto.Conn
+	opts         *ClientOptions
+	state        *stateEmitter
+	stateCh      chan State
+	pending      pendingEmitter
+	queue        *msgQueue
+	auth         *Auth
+	onChannelMsg func(*proto.ProtocolMessage)
+	// This callback is called when we get a response from reconnect request. We
```

Comment:
Totally unimportant, but I'd prefer for "doc comments"'s first word to be the name of the documented thing, [as established](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences), even private ones. Please indulge me :)

```suggestion
	// onReconnectMsg is called when we get a response from reconnect request. We
```